### DISCUSSION TITLE
Deploying web and solid_queue jobs

### DISCUSSION THREAD

**Discussion by ankurp** [basecamp/kamal#655] [Q&A]
I am trying to deploy web and also the background job process on a different host. What is the best way to do so as when I deploy jobs on a different host running a `cmd` it fails the healthcheck as web server is not running.

**Comment by nickhammond** [comment]
Take a look at the [using a custom healthcheck](https://kamal-deploy.org/docs/configuration#using-a-custom-healthcheck) docs, there's also the ability to disable/enable traefik [using-different-roles-for-servers](https://kamal-deploy.org/docs/configuration#using-different-roles-for-servers).
